content/articles/2019-08-lifecycle-0-1-0.Rmarkdown

```diff
@@ -0,0 +1,189 @@
+
+In this post, we'll briefly present the lifecycle workflow. Read the [Get started](http://lifecycle.r-lib.org/articles/lifecycle.html) vignette for a more complete description. Normally you'd call lifecycle functions with the namespace prefix, but for the purposes of illustrating the package we're attaching the package with `library()`:
+
+```{r, message = FALSE}
+library(lifecycle)
+```
+
+
+## Lifecycle stages
+
+The stages for functions and arguments are modelled after the [lifecycle stages for packages](https://www.tidyverse.org/lifecycle/).  There are 4 __development__ stages. Then, if the decision of discontinuing a feature has been made, it goes through 3 __deprecation__ stages.
+
+### Development stages
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-experimental.svg")
+```
+
+This is a new feature that is in the very early stage of development. It is exported so users can start to use it and report feedback, but its interface and/or behaviour is likely to change in the future. It is generally best to avoid depending on experimental features.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-maturing.svg")
+```
+
+The interface and behaviour of a maturing feature has been roughed out, but finer details are likely to change. It still needs more feedback to find the optimal API.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-stable.svg")
+```
+
+A feature is considered stable when the author is happy with its interface and behaviour. Major changes are unlikely, and breaking changes will occur gradually, through a deprecation process.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-questioning.svg")
+```
+
+The author is no longer convinced that the feature is the optimal approach. However, there are no recommended alternatives yet.
+
+
+### Deprecation stages
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-soft-deprecated.svg")
+```
+
+The author is no longer happy with a feature because they consider it sub-optimal compared to some other approach, or simply because they no longer have the time to maintain it. A soft-deprecated feature can still be used without hassle, but users should consider switching to an alternative approach.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-deprecated.svg")
+```
+
+The feature is likely to be discontinued in the next major release. Users should switch to an alternative approach as soon as possible.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-defunct.svg")
+```
+
+The feature can no longer be used. A defunct function is still exported, and a defunct argument is still part of the signature. This way an informative error can be thrown.
+
+You can insert the lifecycle badges anywhere in your documentation. First import the badges in your package:
```

Comment:
Maybe add a subheader before this section indicating that this is the how-to part?